supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesRuntimeAttributes.scala

```diff
@@ -21,28 +18,17 @@ case class JesRuntimeAttributes(cpu: Int,
                                 bootDiskSize: Int,
                                 memory: MemorySize,
                                 disks: Seq[JesAttachedDisk],
+                                /*
+ TODO: There are some interesting comments within NonPreemptibleJesPipelineInfoBuilder around dockerImage, possibly
```

Comment:
Despite what git blame says I'm 99% sure I wrote that. It both sounds like me and I vaguely remember it. IIRC it was more of a "this could potentially happen" thing, part of my love of type safety.